cherab/tools/observers/bolometry.py

```diff
@@ -513,181 +386,25 @@ def calculate_etendue(self, ray_count=10000, batches=10, print_results=False):
                 while True:
 
                     # Find the next intersection point of the ray with the world
-                    intersection = bolometer_world.hit(CoreRay(origin, direction))
+                    intersection = world.hit(CoreRay(origin, direction))
 
                     if intersection is None:
                         passed += 1 * path_weight
                         break
 
                     elif isinstance(intersection.primitive.material, NullMaterial):
                         hit_point = intersection.hit_point.transform(intersection.primitive_to_world)
-                        origin = hit_point + direction * 1E-9
+                        origin = hit_point + direction * 1E-4
```

Comment:
Again, magic number. Can this be made more robust?

Reply:
Same as above